ProjectName/src/store/messagesSlice/index.ts

```diff
@@ -0,0 +1,12 @@
+import {useAppDispatch, useAppSelector} from '../hooks';
+import {deleteMes, MessagesSelector} from './messagesSlice';
+
+export const useMessages = () => {
+  const MessagesState = useAppSelector(MessagesSelector);
```

Comment:
Please, let's stick to naming conventions commont to JS world
```ts
variables,
someFunctions, // <= please pay attention functions should start with verbs
someObject,
ClassName,
SOME_CONSTANT // <= this is not what you declare using const, but something you expect as persistend data (such as MOCKS, SOME_PERSISTENT_TEXT, SOME_NUMBER_CONSTANTS)
Types, Interfaces, Enums
```

Reply:
So applying given rule here we should have something like
`const messagesState = useAppSelector(messagesSeletor)`